frontend/src/modules/dealerships/DealershipLocations.vue

```diff
@@ -0,0 +1,345 @@
+                :draggable="true"
+                :options="{
+                  fillColor: zone.fillColor,
+                  fillOpacity: 0.5,
+                  strokeWeight: 0,
+                  strokeColor: 'blue',
+                }"
+              />
+              <GmapMarker
+                :key="index"
+                v-for="(zone, index) in zones"
+                :position="zone.center"
+                :clickable="true"
+                :visible="selectedZone == zone._id"
+              />
+            </GmapMap>
+          </CCol>
+        </CRow>
+        
+        <CDataTable
+          :items="zones"
+          :fields="fields"
+          class="mt-3"
+          clickable-rows
+          @row-clicked="setSelectedZone"
+          :items-per-page="5"
+          pagination
+        >
+          <template #details="{ item }">
+            <CCollapse :show="item._id == selectedZone" :duration="200">
+              <location-details :zone="item" @close="closeExpanded" @closeAndSave="closeExpandedAndSave"/>
+            </CCollapse>
+          </template>
+        </CDataTable>
+        <CButton color="primary" @click="addingNewZone = true">Add a location zone</CButton>
+        <!-- <CRow>
+          <CCol>
+            
+
+            <CListGroup
+              class="mt-3 mb-3"
+              v-if="zones.length > 0 && !addingNewZone"
+            >
+              <div v-for="(zone, index) in zones" :key="zone._id">
+                <CListGroupItem
+                  tag="button"
+                  :id="zone._id"
+                  :active="selectedZone && zone._id == selectedZone._id"
+                  @click="setSelectedZone(zone, index)"
+                >
+                  {{ zone.name }}
+                </CListGroupItem>
+              </div>
+            </CListGroup>
+            <CButton
+              color="primary"
+              @click="addingNewZone = true"
+              v-if="!addingNewZone"
+              >Add New Zone</CButton
+            >
+            <CForm @submit.prevent="submitNewZone" v-if="addingNewZone">
+              <CInput
+                label="Name"
+                placeholder="Unique name for the zone"
+                v-model="newZoneName"
+                required
+              />
+              <CTextarea
+                label="Description (optional)"
+                placeholder="Enter the zone's description"
+                rows="5"
+                v-model="newZoneDescription"
+              />
+              <CButton
+                color="secondary"
+                @click="
+                  () => {
+                    path = [];
+                    addingNewZone = false;
+                  }
+                "
+                >Cancel</CButton
+              >
+              <CButton color="primary" type="submit">Add Zone</CButton>
+              <CButton color="danger" @click="undoLastPoint">Undo</CButton>
+            </CForm>
+          </CCol>
+        </CRow> -->
```

Comment:
Commented code should be removed

Reply:
Addressed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/87/commits/b3016ee9df8271d80dc6fc49195c95de810e1398)